pkg/arvo/lib/chat-store.hoon

```diff
@@ -0,0 +1,210 @@
+/-  *chat-store
+^?
+|%
+++  enjs
+  =,  enjs:format
+  |%
+  ::
+  ++  letter
+      |=  =^letter
+      ^-  json
+      =,  enjs:format
+      ?-  -.letter
+          %text
+      (frond %text s+text.letter)
```

Comment:
```suggestion
  ++  letter
    |=  =^letter
    ^-  json
    =,  enjs:format
    ?-  -.letter
        %text
      (frond %text s+text.letter)
```

Looks like this file in its entirety of `+enjs` is using 4 instead of 2 spaces for its indentation. (ie the `?:`s below)